data/card_catalogue/unique-1.json

```diff
@@ -299,16 +209,6 @@
       "image": "assets/core/card/illustration/Smoke Bomb.png"
     }
   },
-  {
-    "title": "Phoenix Reborn",
-    "upgrade": "",
-    "tier": "red",
-    "details": {
-      "rules": "The next time you're unconscious, return to life with 20 HP. For each card you discarded from your hand after going unconscious, draw another of the same rarity from the deck.",
-      "flavour": "What he saw on the other side changed him.",
-      "image": "assets/core/card/illustration/Phoenix Reborn.png"
-    }
-  },
```

Comment:
I don't think mentioning the unconscious state is going to be a good pattern for cards going forward. It could occur too long after activation, it'd be a hassle to keep track of

I tried to think up a new version of "become someone else" but didn't come up with anything particularly good. Might revisit the idea again at some point.

Reply:
I'm in favor of scrapping the unconsciousness mechanics, I think it's best for the game to be designed as if you die at 0 HP. 

I think we should avoid canonizing anything beyond "zero HP = new character" in the cards so the DM can handle deaths as they see fit for their campaign.

Reply:
I like that as a concept, I just haven't seen it play out well in gameplay. Deaths have thusfar been a letdown.

I'm up to give it another try though, now that we've got new players and collectively have more DM experience

Reply:
So far in cardmaster we haven't really designed any levels in which the players could reasonably lose, since it's hard to estimate just how broken the cards are

Reply:
I'll separate out the discussion on that into an issue: https://github.com/elliottomlinson/rpcg/issues/83